### PR TITLE
feat: Update credentials for multi arch images

### PR DESCRIPTION
**Pull request by LiteBrick204**
# Changes

This is for #2223
/kind feature
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/resources/images/docker-multi-arch-template.yaml [hunk 1]
```diff
@@ -89,8 +89,9 @@ spec:
             for tag in ${TAGS}; do DESTINATIONS="${DESTINATIONS} --tag ${IMAGE_URL}:${tag}"; done
             IFS=$OIFS
 
-            gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-            gcloud auth configure-docker
+            IMAGE_REGISTRY_USER=tekton-robot
+            IMAGE_REGISTRY_TOKEN=$(cat "$IMAGE_REGISTRY_CREDENTIALS")
+            docker login -u ${IMAGE_REGISTRY_USER} --password-stdin ${IMAGE_REGISTRY_TOKEN}
```
Comment (AlanGreene) [inline tekton/resources/images/docker-multi-arch-template.yaml:94]: Same feedback as on https://github.com/tektoncd/plumbing/pull/2327 re. correct use of `--password-stdin`

File: tekton/resources/images/docker-multi-arch-template.yaml [hunk 2]
```diff
@@ -89,8 +89,8 @@ spec:
             for tag in ${TAGS}; do DESTINATIONS="${DESTINATIONS} --tag ${IMAGE_URL}:${tag}"; done
             IFS=$OIFS
 
-            gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-            gcloud auth configure-docker
+            IMAGE_REGISTRY_USER=tekton-robot 
+            docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin < ${"$IMAGE_REGISTRY_CREDENTIALS"}
```
Comment (AlanGreene) [inline tekton/resources/images/docker-multi-arch-template.yaml:93]: This causes an error due to unterminated string. The following works:

```suggestion
            docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin < "${IMAGE_REGISTRY_CREDENTIALS}"
```

**Review by AlanGreene — CHANGES_REQUESTED** [review]

**Review by AlanGreene — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]

**Review by afrittoli — COMMENTED** [review]
/lgtm

**Pull request merged**